updates/seed_callingcodes.php

```diff
@@ -0,0 +1,247 @@
+            'SK' => ['name' => 'SLOVAKIA', 'code' => '421'],
+            'SL' => ['name' => 'SIERRA LEONE', 'code' => '232'],
+            'SM' => ['name' => 'SAN MARINO', 'code' => '378'],
+            'SN' => ['name' => 'SENEGAL', 'code' => '221'],
+            'SO' => ['name' => 'SOMALIA', 'code' => '252'],
+            'SR' => ['name' => 'SURINAME', 'code' => '597'],
+            'ST' => ['name' => 'SAO TOME AND PRINCIPE', 'code' => '239'],
+            'SV' => ['name' => 'EL SALVADOR', 'code' => '503'],
+            'SY' => ['name' => 'SYRIAN ARAB REPUBLIC', 'code' => '963'],
+            'SZ' => ['name' => 'SWAZILAND', 'code' => '268'],
+            'TC' => ['name' => 'TURKS AND CAICOS ISLANDS', 'code' => '1649'],
+            'TD' => ['name' => 'CHAD', 'code' => '235'],
+            'TG' => ['name' => 'TOGO', 'code' => '228'],
+            'TH' => ['name' => 'THAILAND', 'code' => '66'],
+            'TJ' => ['name' => 'TAJIKISTAN', 'code' => '992'],
+            'TK' => ['name' => 'TOKELAU', 'code' => '690'],
+            'TL' => ['name' => 'TIMOR-LESTE', 'code' => '670'],
+            'TM' => ['name' => 'TURKMENISTAN', 'code' => '993'],
+            'TN' => ['name' => 'TUNISIA', 'code' => '216'],
+            'TO' => ['name' => 'TONGA', 'code' => '676'],
+            'TR' => ['name' => 'TURKEY', 'code' => '90'],
+            'TT' => ['name' => 'TRINIDAD AND TOBAGO', 'code' => '1868'],
+            'TV' => ['name' => 'TUVALU', 'code' => '688'],
+            'TW' => ['name' => 'TAIWAN, PROVINCE OF CHINA', 'code' => '886'],
+            'TZ' => ['name' => 'TANZANIA, UNITED REPUBLIC OF', 'code' => '255'],
+            'UA' => ['name' => 'UKRAINE', 'code' => '380'],
+            'UG' => ['name' => 'UGANDA', 'code' => '256'],
+            'US' => ['name' => 'UNITED STATES', 'code' => '1'],
+            'UY' => ['name' => 'URUGUAY', 'code' => '598'],
+            'UZ' => ['name' => 'UZBEKISTAN', 'code' => '998'],
+            'VA' => ['name' => 'HOLY SEE (VATICAN CITY STATE)', 'code' => '39'],
+            'VC' => ['name' => 'SAINT VINCENT AND THE GRENADINES', 'code' => '1784'],
+            'VE' => ['name' => 'VENEZUELA', 'code' => '58'],
+            'VG' => ['name' => 'VIRGIN ISLANDS, BRITISH', 'code' => '1284'],
+            'VI' => ['name' => 'VIRGIN ISLANDS, U.S.', 'code' => '1340'],
+            'VN' => ['name' => 'VIET NAM', 'code' => '84'],
+            'VU' => ['name' => 'VANUATU', 'code' => '678'],
+            'WF' => ['name' => 'WALLIS AND FUTUNA', 'code' => '681'],
+            'WS' => ['name' => 'SAMOA', 'code' => '685'],
+            'XK' => ['name' => 'KOSOVO', 'code' => '381'],
+            'YE' => ['name' => 'YEMEN', 'code' => '967'],
+            'YT' => ['name' => 'MAYOTTE', 'code' => '262'],
+            'ZA' => ['name' => 'SOUTH AFRICA', 'code' => '27'],
+            'ZM' => ['name' => 'ZAMBIA', 'code' => '260'],
+            'ZW' => ['name' => 'ZIMBABWE', 'code' => '263']
```

Comment:
Why are the name values required? Also could you add an inline comment referencing where you got the values from? Finally, I think the column in the DB should be `calling_code`, not `callingcode`

Reply:
I changed it to calling_code, i used to export csv to php array then i used names to return to my code then i reused them here, if you want i can remove them later.